src/DataTablesMixin.js

```diff
@@ -34,21 +38,47 @@ import Promise from 'bluebird'
  *   }
  * ```
  *
+ * The `useDataTablesCSS`, `useDataTablesURL`, and `useDataTablesEnableScript` are needed for enabling additional
+ * extensions, e.g. to use Datatables.select:
+ * ```
+ *   class MyView extends DataTablesMixin(EditController) {
+ *       constructor(options={}) {
+ *          options.useDataTablesURL = [
+ *            "//cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js",
+ *            "//cdn.datatables.net/select/1.2.7/js/dataTables.select.min.js"
+ *          ]
+ *          options.useDataTablesCSS = [
+ *            "//cdn.datatables.net/1.10.19/css/jquery.dataTables.min.css",
+ *            "//cdn.datatables.net/select/1.2.7/css/select.dataTables.min.css"
+ *          ]
+ *          // this file in your project would include `$('.datatable).DataTable({select: true})` etc
+ *          options.useDataTablesEnableScript = __dirname+"/components/my-datatables-enable.js"
+ *          super(options)
+ *      }
+ *   }
+ * 
+ * ```
+ * 
  */
 
 let DataTablesMixin = (superclass) => class extends(superclass) {
   constructor(opts={}) {
     super(opts)
 
     this.useDataTablesAjax = opts.useDataTablesAjax || false
-    this.useDataTablesCSS = true
+    this.useDataTablesURL = opts.useDataTablesURL || '//cdn.datatables.net/1.10.16/js/jquery.dataTables.min.js'
+    if (!isArray(this.useDataTablesURL)) {
+      this.useDataTablesURL = [this.useDataTablesURL]
+    }
+    this.useDataTablesCSS = ['//cdn.datatables.net/1.10.16/css/jquery.dataTables.min.css']
```

Comment:
In switching this from boolean to array (per the comment change above), it looks like old code that set the `opts.useDataTablesCSS` to `true` would get bit down in the web-controller-datatables-list.ejs 
unless I'm missing something.

Reply:
You're right, I'll put a guard on that as with `useDataTablesURL`. I don't believe there are any uses of that in the wild, but best to be defensive.